f31-py37/Pipfile

```diff
@@ -0,0 +1,13 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+thamos = "*"
+micropipenv = {extras = ["toml"], version = "*"}
+
+[dev-packages]
+
+[requires]
+python_version = "3.7"
```

Comment:
kept the Pipfile and Pipfile.lock for easy maintenance. 
can be removed if feels like an unnecessary addition.

Reply:
I think it can be removed. You can directly pass `Pipfile.lock` to micropipenv, without ``-deploy`` it will not need Pipfile.

Reply:
i will remove the pipfile and pipfile.lock
and let stay with the requirements.txt, as it is easy to use in other build tools.

Reply:
i have made the changes and only kept the requirements.txt.